### PR TITLE
Added legacy config

### PR DESCRIPTION
**Pull request by leoafarias**
Generates legacy fvm_config.json file to better support tools that need to migrate

### CODE REVIEW

File: lib/src/models/project_model.dart [hunk 1]
```diff
@@ -40,16 +40,15 @@ class Project {
...
     final legacyConfigFile = _legacyFvmConfigPath(path);
 
+    final config = ProjectConfig.loadFromPath(configFile);
+
     // Used for migration of config files
     final legacyConfig = ProjectConfig.loadFromPath(legacyConfigFile);
 
-    if (legacyConfig != null) {
+    if (config == null && legacyConfig != null) {
       legacyConfig.save(configFile);
```
Comment (jibbers42) [inline lib/src/models/project_model.dart:49]: Does this need to load the new config?

Comment (leoafarias) [reply lib/src/models/project_model.dart:49]: Yes, this is the migration of the new config that does not exist. It creates a new one based on the old one.

Comment (jibbers42) [reply lib/src/models/project_model.dart:49]: Yes, but `config` is still null right, doesn't it need to be loaded after migration?

**Review by jibbers42 — APPROVED** [review]
LGTM

Thanks for this!

**Pull request merged**